ocp_resources/ServiceMeshMemberRoll.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please rename the file to keep the same convention as other resources.

Reply:
Missed it, updated. Thanks:)